### PR TITLE
Bug 1373045, added information about `oc policy can-i --list`

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1373045

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@deads2k PTAL. I see in the [test case](http://tcms-openshift.rhcloud.com/case/5337/) that various scopes are included. Do you think that is useful information to include? If so, what is a use case for each? (It's currently not very intuitive to me.) Thanks!

**Comment by ahardin-rh** [comment]
https://github.com/openshift/openshift-docs/pull/2631 includes information on scopes, so I will give that content some love and point to it from here.

**Comment by ahardin-rh** [comment]
@deads2k @liggitt PTAL

@adellape @bfallonf please peer review.

Thanks

**Comment by bfallonf** [comment]
@ahardin-rh I think this is largely fine. The only thing I would comment on is the use of "verbs". I'm not sure if every reader will fully understand what a verb is (I'm not even sure if I'm sure). Maybe a link to the below somewhere?

https://docs.openshift.com/enterprise/3.2/cli_reference/basic_cli_operations.html#application-modification-cli-operations

**Comment by ahardin-rh** [comment]
@bfallonf Thanks! If the definition of verbs is the stumbling block, then I should point users to the section earlier on that page that [defines these actions](https://docs.openshift.com/enterprise/3.2/architecture/additional_concepts/authorization.html#action) vs. pointing to the `oc describe` command. I made an update to accomplish that. :beers:

**Comment by bfallonf** [comment]
@ahardin-rh Works for me! :+1:

File: architecture/additional_concepts/authorization.adoc [hunk 1]
```diff
@@ -681,3 +681,38 @@ delimited list of blocks in the format of `<start>/<length` or `<start>-<end>`.
...
+The output will provide a full list.
+
+To determine if you can perform specific xref:action[verbs], run:
+
+----
+$ oc policy can-i <verb> <resource>
+----
+
+xref:../../admin_guide/scoped_tokens.adoc#admin-guide-scoped-tokens-user-scopes[User
+scopes] can provide more information about a given user. For example:
```
Comment (wjiangjay) [inline architecture/additional_concepts/authorization.adoc]: "about a given user"? not "about a given scope"?

Comment (ahardin-rh) [inline architecture/additional_concepts/authorization.adoc]: @wjiangjay Thanks! I made this edit.

**Comment by ahardin-rh** [comment]
Approved by QE in BZ

**Comment by ahardin-rh** [comment]
[rev_history]
|xref:../architecture/additional_concepts/authorization.adoc#architecture-additional-concepts-authorization[Additional Concepts -> Authorization]
|Added a  xref:../architecture/additional_concepts/authorization.adoc#authorization-determining-what-you-can-do-as-an-authenticated-user[new section] about `oc policy can-i --list`.
%

**Comment by vikram-redhat** [comment]
@ahardin-rh - this content is no longer present in 3.5 and while doing the Online Pro release and working on this PR (https://github.com/openshift/openshift-docs/pull/4620/), I noticed that this file caused a conflict for the 3.4 branch. I decided to remove it because it was not in 3.5, but not really sure if that was the right thing to do.  I have tried to look at the history, but I can't seem to figure out if this was only a 3.3 thing and should or shouldn't apply to 3.4 and 3.5.

Can you please check? If this does apply to 3.4 and 3.5, then you can cherry pick it or create a new PR to add it back in.

**Comment by ahardin-rh** [comment]
@deads2k ^^ Should this content persist in OCP 3.4 and 3.5 docs?

**Comment by ahardin-rh** [comment]
@deads2k @liggitt Does this content belong in the OCP 3.4 and 3.5 docs?

**Comment by deads2k** [comment]
> @deads2k @liggitt Does this content belong in the OCP 3.4 and 3.5 docs?

Yes.  If I'm reading the tags right, it merged in time for 1.3.  https://github.com/openshift/origin/commit/c37d75d72e9a4bda28ac2b3961654a119b76f9e7

**Comment by ahardin-rh** [comment]
@deads2k @vikram-redhat Thanks! This is fixed via https://github.com/openshift/openshift-docs/issues/4815 and https://github.com/openshift/openshift-docs/issues/4816

**Pull request merged**